Src/FluentAssertions/Equivalency/EquivalencyAssertionOptions.cs

```diff
@@ -38,6 +38,17 @@ public EquivalencyAssertionOptions<TExpectation> Excluding(Expression<Func<TExpe
             return this;
         }
 
+        /// <summary>
+        /// Excludes the specified (nested) member from the structural equality check.
+        /// Allows to navigate deeper by using the returned <see cref="EquivalencyAssertionOptionsBuilder{TExpectation, TNext}"/>.
+        /// </summary>
+        public EquivalencyAssertionOptionsBuilder<TExpectation, TNext> Excluding<TNext>(Expression<Func<TExpectation, IEnumerable<TNext>>> expression)
```

Comment:
🤔 Why do we need another overload?

Reply:
I can not use the existing signature, because it does not have the `TNext` generic parameter. I can also not add the `TNext` to the existing signature, because I need to have `IEnumerable<TNext>` in the return parameter of the `Expression`.

Reply:
Clear!